Cargo.toml

```diff
@@ -91,6 +91,7 @@ memoize = { version = "0.4", features = ["full"] }
 vbs = "0.1"
 clap = { version = "4", features = ["derive", "env"] }
 url = { version = "2.5.0", features = ["serde"] }
+get_if_addrs = "0.5.3"
```

Comment:
FYI this crate appears to have been abandoned 5 years ago. It looks like it's doing pretty simple logic, perhaps it's worth just copying the necessary functions into this?

Reply:
Good to know, since @QuentinI is going to change this logic let me also tag him.